frontend/src/metabase/query_builder/components/filters/FilterPopover/FilterPopoverHeader.tsx

```diff
@@ -30,6 +30,7 @@ export default function FilterPopoverHeader({
 
   const field = dimension.field();
   const operator = filter.operatorName();
+  const hasEmptyPicker = filter.operator()?.fields.length === 0;
```

Comment:
`hasEmptyPicker` will be `false` if `filter.operator()` is falsy.

Reply:
That's fine, no? If we don't have an operator then we can't have a filter or a picker.

Reply:
Oh, ok, because later it's negated - got it.